native/dlartg.go

```diff
@@ -17,7 +17,8 @@ import "math"
 func (impl Implementation) Dlartg(f, g float64) (cs, sn, r float64) {
 	safmin := dlamchS
 	eps := dlamchE
-	safmn2 := math.Pow(float64(dlamchB), math.Log(safmin/eps))
+	exp := int(math.Log(safmin/eps) / math.Log(float64(dlamchB)) / 2)
+	safmn2 := math.Pow(float64(dlamchB), float64(exp))
```

Comment:
`safmn2 := math.Pow(dlamchB, math.Floor(math.Log(safmin/eps) / math.Log(dlamchB) / 2)`

`dlamchB` is an untyped number, so it will be float64 for this signature.

Is there any reason not to have values like this as package global?